JS: Arrow Functions

SergioGutierrezVillalba opened the pull request:
Basic arrow function tutorial.

ronreiter commented:
@SergioGutierrezVillalba 
Hey, can you remove the .vscode/settings.json file and add it to the .gitignore?
Also - can you explain the latin-1 change?

SergioGutierrezVillalba commented:
I changed utf8 to latin-1, because I got this error when trying to start the application:
![pythonerror](https://user-images.githubusercontent.com/34506328/78499403-c6864a00-7750-11ea-854a-a7e6b5436986.PNG)

I have added the commit with utf8 codification and vscode removed and added to gitignore.
